developer/src/kmc/src/kmc.ts

```diff
@@ -3,172 +3,44 @@
  * kmc - Keyman Next Generation Compiler
```

Comment:
So... it looks like this file is mostly just split + abstracted to prepare for more complex subcommands?

Looks like the following files were spun off from this:
- activities/buildLdmlKeyboard.ts
- activities/buildTestData.ts
- commands/build.ts
- commands/buildTestData.ts
- util/NodeCompilerCallbacks.ts

I'm not quite fine-tooth-combing this, but it looks mostly like the same code as before, just reorganized.

Just figured I'd document that in case anyone else wants to review... and to confirm that I'm following the changes reasonably well.

Reply:
Yes, pretty much. The command line has changed to have a 'command' as the first parameter, in prep for future commands.

Reply:
yes, `build` and `build-test-data` were previously the default and a separate argument, respectively